app/services/search/query_builders/classified_listing.rb

```diff
@@ -30,6 +32,7 @@ class ClassifiedListing
 
       def initialize(params)
         @params = params.deep_symbolize_keys
+        @params[:published] = DEFAULT_PARAMS[:published] unless @params.key?(:published)
```

Comment:
We can keep this `@params.key?(:published)` here as long as we don't permit `published` as a param for the controller. I don't want people having the ability to override this from the frontend.

Reply:
Ah, I see. I thought we might need that functionality in the future for the API or something. I'll just remove this altogether so it's always `true`. And if we need it later, we can always add it in at that time. How does that sound?

Reply:
Sounds good